src/core/ext/filters/client_channel/resolver/xds/xds_resolver.cc

```diff
@@ -648,13 +648,16 @@ ConfigSelector::CallConfig XdsResolver::XdsConfigSelector::GetCallConfig(
   absl::optional<uint64_t> hash;
   for (const auto& hash_policy : route_action->hash_policies) {
     absl::optional<uint64_t> new_hash;
+    std::string new_hash_string;
```

Comment:
This can go under the `case` statement where it is used.  Just put braces around the body of the `case` statement.  In other words:

```
case XdsRouteConfigResource::Route::RouteAction::HashPolicy::CHANNEL_ID: {
  std::string new_hash_string;
  // ...code...
  break;
}
```

But that having been said, I'm not sure that taking the hash of the address is the best approach to begin with.  Instead, how about just computing a random value at `XdsResolver` instantiation time and then using that value here instead of the address of the `XdsResolver`?  It seems like that would give better distribution.